pkg/gui/controllers/helpers/worktree_helper.go

```diff
@@ -0,0 +1,80 @@
+package helpers
+
+import (
+	"fmt"
+	"github.com/go-errors/errors"
+	"github.com/jesseduffield/lazygit/pkg/commands"
+	"github.com/jesseduffield/lazygit/pkg/commands/models"
+	"github.com/jesseduffield/lazygit/pkg/gui/context"
+	"github.com/jesseduffield/lazygit/pkg/gui/types"
+	"io/fs"
+	"log"
+	"os"
+)
+
+type IWorktreeHelper interface {
+	GetMainWorktreeName() string
+	GetCurrentWorktreeName() string
+}
+
+type WorktreeHelper struct {
+	c        *types.HelperCommon
+	git      *commands.GitCommand
+	contexts *context.ContextTree
+	model    *types.Model
+}
+
+func NewWorktreeHelper(
+	c *types.HelperCommon,
+	git *commands.GitCommand,
+	contexts *context.ContextTree,
+	model *types.Model) *WorktreeHelper {
+	return &WorktreeHelper{
+		c:        c,
+		git:      git,
+		contexts: contexts,
+		model:    model,
+	}
+}
+
+func (self *WorktreeHelper) GetMainWorktreeName() string {
+	for _, worktree := range self.model.Worktrees {
+		if worktree.Main() {
+			return worktree.Name()
+		}
+	}
+
+	return ""
+}
+
+func (self *WorktreeHelper) IsCurrentWorktree(w *models.Worktree) bool {
+	pwd, err := os.Getwd()
+	if err != nil {
+		log.Fatalln(err.Error())
```

Comment:
Can we add context to this error e.g. 'failed to determine a given worktree is the current worktree'. Given we're not panicking here, it's good to have something that's easy to search for within the codebase to see where it's failing.